docs/manualGrading.md

```diff
@@ -1,10 +1,13 @@
 # Manual Grading
 
-PrairieLearn supports manual grading of questions by downloading a CSV file with student answers and uploading a CSV file with question scores and optional per-question feedback. There is not currently an online web interface for manual grading.
+Some questions may require manual grading, either fully (e.g., freeform essay-type questions) or partially (e.g., coding style, or "show your work" components). PrairieLearn currently supports two methods of manual grading:
+
+- Course staff may download a CSV file with student answers, perform any grading tasks offline, and upload a CSV file with question scores and optional per-question feedback.
+- An online web interface is available for individual grading directly into the assessment question.
 
 ## Configuring a question for manual grading
 
-The [`info.json` file](question.md#question-infojson) in the question should set `"gradingMethod": "Manual"`, like this:
+For questions that require full manual grading, without any auto-grading functionality, the [`info.json` file](question.md#question-infojson) in the question may be set `"gradingMethod": "Manual"`, like this:
```

Comment:
> To check my own understanding: will setting `manualPoints` without also setting `autoPoints` function basically the same as `"gradingMethod": "Manual"`?

Yes, that is correct, though setting the grading method to `Manual` still can be used to:
* set the default if `points` is used instead of `manualPoints`;
* change the default behaviour of the question preview page.

> If it can, I'm wondering if we should break up the docs into the "preferred" manual grading flow (all your great new work) and the "legacy"/"advanced" manual grading workflow. If someone is just skimming this page, I'd want them to hit the modern process and not waste time with `gradingMethod`/`singleVariant`.

That can be done, though all example course questions continue to have references to points and the manual grading method. Should that be kept? Also, there's still use for the suggestion to use `singleVariant`.

Reply:
Updated docs. Grading method has been removed from this page, left only in other pages. I also changed the assessment configuration docs.